examples/js/shaders/AfterimageShader.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @author HypnosNova / https://www.threejs.org.cn/gallery/
+ *
+ * Afterimage shader
+ * I created this effect inspired by a demo on codepen:
+ * https://codepen.io/brunoimbrizi/pen/MoRJaN?page=1&
+ */
+
+THREE.AfterimageShader = {
+
+	uniforms: {
+
+		"damp": { value: 0.96 },
+		"tOld": { value: null },
+		"tNew": { value: null }
+
+	},
+
+	vertexShader: [
+
+		"varying vec2 vUv;",
+
+		"void main() {",
+
+			"vUv = uv;",
+			"gl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );",
+
+		"}"
+
+	].join( "\n" ),
+
+	fragmentShader: [
+
+		"uniform float damp;",
+
+		"uniform sampler2D tOld;",
+		"uniform sampler2D tNew;",
+
+		"varying vec2 vUv;",
+		
+		"float when_gt( float x, float y ) {",
+
+			"return max( sign( x - y ), 0.0 );",
+
+		"}",
+
+		"void main() {",
+
+			"vec4 texelOld = texture2D( tOld, vUv );",
+			"vec4 texelNew = texture2D( tNew, vUv );",
+			
+			"texelOld *= damp;",
+			
+			"texelOld.r *= when_gt( texelOld.r, 0.2 );",
+			"texelOld.g *= when_gt( texelOld.g, 0.2 );",
+			"texelOld.b *= when_gt( texelOld.b, 0.2 );",
+			"texelOld.a *= when_gt( texelOld.a, 0.2 );",
```

Comment:
Try this, instead
```glsl
"vec4 when_gt( vec4 x, float y ) {",
...
"texelOld *= when_gt( texelOld, 0.2 );",
```

Reply:
thanks, i've approved the code and fixed the effect problem. Now it looks good without wireframe

Reply:
@WestLangley thanks, i've approved the code and fixed the effect problem. Now it looks good without wireframe.